src/tests/functional/plugin/shared/src/behavior/ov_infer_request/batched_tensors.cpp

```diff
@@ -37,7 +37,6 @@ void OVInferRequestBatchedTests::SetUp() {
 
 void OVInferRequestBatchedTests::TearDown() {
     if (m_need_reset_core) {
-        ie->set_property({ov::cache_dir()});
```

Comment:
these calls are valid. Why do we replace it?
maybe we need to fix Core instead?

is it enough to apply this fix https://github.com/openvinotoolkit/openvino/pull/24018/files#r1572011425 ?

Reply:
Per my knowledge, the request here is to set property through compile_model only.
@sbutnari This is test case, so it should test all the possibility. If this is an example code, we can show user to use compile_model  for properties setting only.
The recommendation is to keep this test case to test ov::Core::sert_property(), properties setting with compile_model only should already be tested by other test cases.

Reply:
> Per my knowledge, the request here is to set property through compile_model only.

we can ignore such request, because we need to test all approach in tests, but not only one which works on NPU